BUILD

```diff
@@ -37,9 +37,6 @@ cc_library(
         "-DHAVE_RWLOCK",
         "-DGFLAGS_INTTYPES_FORMAT_C99",
     ],
-    includes = [
```

Comment:
I suppose a `include` subdirectory is implicitly added by Bazel and this is just obsolete, but not wrong?